### DISCUSSION TITLE
Is it possible to use leptos with winit?

### DISCUSSION THREAD

**Discussion by ghost** [leptos-rs/leptos#549] [Q&A]
Just curious if we are able to use leptos with winit (window management) to be able to create desktop applications

**Comment by gbj** [comment]
Honestly I'm completely unfamiliar with winit so probably can't give a useful answer — probably worth a quick look at the `gtk` example in the repo to see what using Leptos with native GUI looks like. It's lacking all the useful rendering stuff we are doing in the DOM but the reactivity works fine to drive changes.

**Reply by gbj** [reply]
So you mean to have a webview with a Leptos UI rendered using the usual Leptos DOM API, running inside a window created/controlled by winit? Sure, you'd have to do some work to glue these things together and like I said, I have no experience with winit but I'm sure it's possible. This is kind of the Tauri approach I guess?

If you mean a winit window with some kind of native GUI rendering inside it, then -- see above about the GTK example. There's not a full-feature native GUI library built using Leptos.
